routes/index.js

```diff
@@ -909,6 +884,54 @@ router.get('/api/v1/equipment', (req, res, next) => {
   });
 });
 
+/* GET floor awards value */
+router.post('/api/v1/awardsOnly', (req, res, next) => {
```

Comment:
Line 888: it has router.post() but this should be router.get(), but the stuff in here is definitely a get.